app/views/smart_proxies/plugins/_dhcp_subnets.html.erb

```diff
@@ -0,0 +1,69 @@
+<ul id="proxy-dhcp-tab" class="nav nav-tabs nav-tabs-pf">
+  <li class="active"><a href="#dhcp_general"><%= _('General') %></a></li>
+  <% subnets.each do |subnet| %>
+    <li><a href="#<%= subnet.network.parameterize %>"><%= subnet_label subnet %></a></li>
+  <% end %>
+</ul>
+<div id="proxy-dhcp-tab-content" class="tab-content">
+  <div class="tab-pane active in" id="dhcp_general">
+    <div class="row">
+      <%= show_feature_version('dhcp') %>
+    </div>
+
+    <div class="container-fluid container-cards-pf">
+      <div class="row row-cards-pf">
+        <% subnets.in_groups_of(2).each do |subnets| %>
+          <% subnets.each do |subnet| %>
+            <div class="col-md-4">
+              <div class="card-pf card-pf-aggregate-status card-pf-accented">
+                <h2 class="card-pf-title">
+                  <a href="#<%= subnet.network.parameterize %>">
+                    <%= subnet_label(subnet) %>
+                    <span class="card-pf-aggregate-status-count">
+                      <%= Host.joins(:interfaces => :subnet).where(:subnets => {:dhcp_id => @smart_proxy.id, :network => subnet.network }).count %>
+                    </span>
+                    <%= _('Hosts') %>
```

Comment:
I think all figures, like # of hosts should go below as `aggregate-status-notifications`